docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -123,6 +123,13 @@ type:: boolean
 
 Should volume expansion be allowed with this storage class.
 
+=== `retainpolicy`
+
+[horizontal]
+type:: string
+
```

Comment:
default should be added here

Reply:
It is an optional parameter, will remove it from the defaults.